src/MIDebugEngine/AD7.Impl/AD7Expression.cs

```diff
@@ -65,6 +66,22 @@ int IDebugExpression2.EvaluateAsync(enum_EVALFLAGS dwFlags, IDebugEventCallback2
 
         // This method evaluates the expression synchronously.
         int IDebugExpression2.EvaluateSync(enum_EVALFLAGS dwFlags, uint dwTimeout, IDebugEventCallback2 pExprCallback, out IDebugProperty2 ppResult)
+        {
+            return EvaluateSync(dwFlags, DAPEvalFlags.NONE, dwTimeout, pExprCallback, out ppResult);
+        }
+
+        #endregion
+
+        #region IDebugExpressionDAP
+
+        int IDebugExpressionDAP.EvaluateSync(enum_EVALFLAGS dwFlags, DAPEvalFlags dapFlags, uint dwTimeout, IDebugEventCallback2 pExprCallback, out IDebugProperty2 ppResult)
+        {
+            return EvaluateSync(dwFlags, dapFlags, dwTimeout, pExprCallback, out ppResult);
+        }
+
+        #endregion
+
+        private int EvaluateSync(enum_EVALFLAGS dwFlags, DAPEvalFlags dapFlags, uint dwTimeout, IDebugEventCallback2 pExprCallback, out IDebugProperty2 ppResult)
```

Comment:
The pattern with the internal implementation is to add Internal to the name like `EvaluateSyncInternal`